packages/playground/windows/playground/MainPage.cpp

```diff
@@ -51,6 +51,10 @@ void MainPage::OnLoadClick(
   }
   host.InstanceSettings().JavaScriptBundleFile(bundleFile);
 
+  winrt::Microsoft::ReactNative::QuirkSettings::SetEnableFabric(
+      host.InstanceSettings(), x_UseFabric().IsChecked().GetBoolean());
+  ReactRootView().ExperimentalUseFabric(x_UseFabric().IsChecked().GetBoolean());
+
```

Comment:
why is it necessary to set both? wouldn't it be enough to just set it in the react root view?

Reply:
One is a setting on the instance, which causes the instance to create some internal infrastructure that is needed to be in the instance to load fabric root views.  In particular it creates the fabric ui manager.  I could just have that always be created if building with the USE_FABRIC flag. -- I'll look at removing this setting then.